### PR TITLE
Add liberty nature using the configure explorer menu

### PR DESCRIPTION
**Pull request by mezarin**
Fixes: #197

### CODE REVIEW

**Review by scottkurz — COMMENTED** [review]
I'm not sure this is related to your code change or not but let me just ask since I'm thinking about it... did we test the case where the user manually edits the .project file adding the nature?  

Does that trigger the resource change listener to update the dashboard?

File: bundles/io.openliberty.tools.eclipse.ui/plugin.xml [hunk 1]
```diff
@@ -304,7 +344,7 @@
                       <iterate>
                           <and>
                               <instanceof value="org.eclipse.core.runtime.IAdaptable"/>
-                              <test property="org.eclipse.debug.ui.projectNature" value="io.openliberty.tools.eclipse.ui.libertyNature"/>
+                              <test forcePluginActivation="true" property="org.eclipse.debug.ui.projectNature" value="io.openliberty.tools.eclipse.ui.libertyNature"/>
```
[the remark this answers is not in the document]
Comment (mezarin) [reply bundles/io.openliberty.tools.eclipse.ui/plugin.xml:347]: This is just pre-work for removing EarlyStartupRegistration (thorough testing needed). Per the [doc](https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fextension-points%2Forg_eclipse_ui_handlers.html), that flag will trigger the activation of the plugin when evaluated: "... If forcePluginActivation is set to true and the evaluation context used to evaluate this expression support plug-in activation then evaluating the property will result in activating the plug-in defining the tester."

[the remark this answers is not in the document]
Comment (mezarin) [reply bundles/io.openliberty.tools.eclipse.ui/plugin.xml:347]: In case of a manual update to .project, a refresh is needed.

**Review by scottkurz — APPROVED** [review]
Looks good.  I like the Open Liberty icon in there as a nice touch.  Assuming the checks run we can merge.

You were going to open a follow-up issue for tests?

**Comment by mezarin** [comment]
Yep. That will be handled with issue #232

**Pull request merged**